src/components/accordions/Timeline.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { mdiLeaf } from "@mdi/js"
+import {
+  Timeline,
+  TimelineProps
+}                  from "components/accordions/Timeline"
+import { Icon }    from "components/Icon"
+import React       from "react"
+
+export default {
+  title:     "accordions/Timeline",
+  component: Timeline,
+  argTypes:  {}
+}
+
+const data = [
+  { content:  "hello world",
+    dateTime: "2023-10-21, 9:50 PM",
+    icon:     <Icon size={"unset"} svg={mdiLeaf} />
+  },
+  { content: "good bye world", dateTime: "2023-10-21, 9:50 PM" }
```

Comment:
Can you add a data entry where the content has a new line character?

Reply:
new line character (\n) isn't working. May be zendesk garden library takes care of that based on the content and width of the container.

Reply:
Moreso, I'm referring to when the user adds their own blank line in between content, like so:

```
Bla bla bla bla

bla bla
```

That's something you'd have to account for in the actual component. Adding a css attribute for respecting `white-space` to the Text component wrapping the content should suffice.